backend/conferences/models/conference_voucher.py

```diff
@@ -49,6 +50,8 @@ class VoucherType(models.TextChoices):
         help_text=_("When the email was last sent"), blank=True, null=True
     )
 
+    objects = ConferenceVoucherQuerySet().as_manager()
+
```

Comment:
Not the purpose of this PR, but why this configuration is hardcoded below?? 
if we change our mind, we need to update the code... what if conferences have different discounts? 🤔